lib/features/mailbox_dashboard/domain/model/unread_spam_emails_response.dart

```diff
@@ -0,0 +1,14 @@
+
+import 'package:equatable/equatable.dart';
+import 'package:jmap_dart_client/jmap/mail/email/email.dart';
+
+class UnreadSpamEmailsResponse with EquatableMixin {
+  final List<Email>? spamEmailList;
```

Comment:
IMO, we will not receive `spamEmailList`. Moreover, we only care for the mailbox with `spam` role